### PR TITLE
[MM 48251]: Add Monthly/Yearly Badge on the Subscription Page

### PR DESCRIPTION
**Pull request by emmyni**
#### Summary

Add a monthly/yearly badge on the subscription page to differentiate between a monthly and yearly subscription plan. The free plan has no badge. 

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-48251

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Screenshots

Free Plan:
image

Monthly Plan:
image

Yearly plan:
image

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by ConorMacpherson — APPROVED** [review]
LGTM! Nice work :)

File: components/admin_console/billing/plan_details/plan_details.tsx [hunk 1]
```diff
@@ -85,6 +86,45 @@ export const PlanDetailsTopElements = ({
         </Badge>
     );
 
+    const monthlyBadge = (
+        <Badge
+            className='RecurringIntervalBadge'
+        >
+            <FormattedMessage
+                id='pricing_modal.monthly'
```
Comment (Afsoon) [inline components/admin_console/billing/plan_details/plan_details.tsx:94]: Nit: I'm not a big fan of using translations from other locations.

**Review by Afsoon — APPROVED** [review]
LGTM @emmyni!. I've just a non blocker comment

**Comment by yasserfaraazkhan** [comment]
@emmyni , the 'monthly' badge is visible.
do we have to get other PR merged to get the 'yearly'?

**Comment by emmyni** [comment]
@yasserfaraazkhan I'll ask someone to turn on the AnnualSubscription feature flag so that the toggle is visible. Basically, this depends on having both the annual toggle PRs and the PRs that make the calls to CWS to convert to yearly plans ( https://github.com/mattermost/mattermost-server/pull/21673 and https://github.com/mattermost/mattermost-webapp/pull/11600). The webapp changes are already included in this branch but we might have to wait for the server PR to be merged to properly convert plans to yearly (and see the yearly badge).

**Review by yasserfaraazkhan — APPROVED** [review]

**Review by annelieseklein — APPROVED** [review]
LGTM! Can we just add a small space before "See how billing works"

**Pull request merged**